flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamConfigChainer.java

```diff
@@ -109,14 +132,14 @@ private void head(StreamOperator<?> headOperator, OperatorID headOperatorID) {
 			tailConfig.setStateKeySerializer(inputSerializer);
 		}
 		tailConfig.setChainIndex(chainIndex);
+		tailConfig.setBufferTimeout(bufferTimeout);
```

Comment:
Why we change the previous timeout 0 here? And it would result in some unit tests failure in travis.

Reply:
I needed the non-zero value for my new test, so that the `OutputFlusher` thread is started.

Yes, I noticed the test failures caused by this. I will think how to address them. Previous behaviour was I think incorrect (`StreamConfigChainer` was ignoring the value passed from `StreamConfig`)